### PR TITLE
data:yassineatik

### PR DESCRIPTION
**Pull request by yassineatik**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7671"></a>

### CODE REVIEW

File: data/yassineatik.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Yassine Atik",
+  "bio": "Fullstack Web Developer : PHP | LARAVEL  ,  REACT | NEXT.JS",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SaraJaoude",
```
Comment (RohanSasne) [inline data/yassineatik.json:7]: Please provide your own twitter handle over here :)

**Review by RohanSasne — CHANGES_REQUESTED** [review]

File: data/yassineatik.json (hunk 1 above)
Comment (yassineatik) [reply data/yassineatik.json]: Thanks @RohanSasne , I have made the necessary changes .

**Review by RohanSasne — APPROVED** [review]
LGTM !!
A Maintainer will soon merge your PR :)

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**